### ISSUE TITLE
tx1-2: Mapping missing for field I.3.3.1 "Additional information can be obtained from" "The above mentioned address"

### ISSUE THREAD

**Issue by csnyulas** [OP-TED/ted-rdf-mapping#258]
Context F21
[comment from N Muric]
> I.3.3.1 No mapping there should be a mapping to the address of the buyer as the ProcurementProcedureInformationProvider for this row

![image](https://user-images.githubusercontent.com/2441046/204917147-ff86574e-4175-476e-b37f-af6d889fe543.png)

E.g. In **034993-2022** there is no mapping for this

Conclusion: Mapping (both technical and conceptual) for field I.3.3.1, corresponding to XPath `CONTRACTING_BODY/ADDRESS_FURTHER_INFO_IDEM` is missing

**Comment by csnyulas** [comment]
Conceptual mapping was added in version 4.1.0

![image](https://user-images.githubusercontent.com/2441046/204925029-619bea01-4ae4-47fe-8510-bc6183d1e3bd.png)

Where in CL1the AgentInRole for such cases will be `epo:ProcurementProcedureInformationProvider`

![image](https://user-images.githubusercontent.com/2441046/204925245-a12527a7-228f-47e8-bdab-afc69a1a34c3.png)

**Comment by muricna** [comment]
1.3.3.1 When there is more than one buyer and therefore more than one address above should there not both addresses be foreseen.

**Comment by csnyulas** [comment]
I opened a separate ticket (#333) for the question raised by @muricna in her last comment, i.e. about how to deal with mapping "The above mentioned address" that appears in field I.3.3.1 (XPath: `CONTRACTING_BODY/ADDRESS_FURTHER_INFO_IDEM`) and I.3.4.2 (XPath: `CONTRACTING_BODY/ADDRESS_PARTICIPATION_IDEM`), when there are **multiple** buyers.

That question is a related, but a separate, issue from what this issue was about, namely the missing technical mapping for the field I.3.3.1. This issue (about creating the appropriate `epo:ProcurementProcedureInformationProvider` and/or `epo:TenderReceiver` instances when `ADDRESS_FURTHER_INFO_IDEM` and/or `ADDRESS_PARTICIPATION_IDEM` XML element is present) has been fixed, as was demonstrated the previous comments, and can be seen when looking at the linked commits.

So, I will mark again this issue as Fixed, and will close it. The question about additional buyer should be dealt with in #333.
